rook/dd/dd-image/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
+
+blockdevices=($BLOCKDEVICES)
+
+i=0
+for blkdev in "${blockdevices[@]}"    
+do
+dd if=/dev/$blkdev of=here bs=1G count=1 oflag=direct 2> "/tmp/dd-${blkdev}"
```

Comment:
this can be done directly: `of=/dev/sdb`

Reply:
I just realized that this is running a read test. We can do both read and write (specifying `of` and `if` directly), and read from `/dev/random` (for writes) and write to `/dev/null` for reads

Reply:
Thanks